Uno.Gallery/Uno.Gallery.Shared/Views/SamplePages/BarometerSamplePage.xaml

```diff
@@ -0,0 +1,64 @@
+﻿<Page x:Class="Uno.Gallery.Views.Samples.BarometerSamplePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:smtx="using:ShowMeTheXAML"
+	  xmlns:local="using:Uno.Gallery"
+	  mc:Ignorable="d">
+
+	<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+		<local:SamplePageLayout IsDesignAgnostic="True">
+			<local:SamplePageLayout.DesignAgnosticTemplate>
+				<DataTemplate>
+					<StackPanel>
+						<smtx:XamlDisplay UniqueKey="BarometerSamplePage_Sample"
+										  smtx:XamlDisplayExtensions.IgnorePath="XX"
```

Comment:
Is this needed?

Reply:
I used it to hide the Xaml code, but still display the c# comment. I am not sure if there is some more elegant solution for this? 🤔